cha/design/sync.tex

```diff
@@ -99,14 +98,38 @@ \section{Conclusion}\label{sec:sync_conclusion}
 \ac{NTP} is both a more mature and widespread protocol, furthermore we were also able to find examples of it being used on Android whereas we could find no Android apps using \ac{PTP}.
 While we have no proof of better or worse performance for the two, the maturity of \ac{NTP} and lackluster use of \ac{PTP} on Android leads us to choose \ac{NTP} as the synchronization method for our basic synchronization milestone.
 
-\begin{table}
-    \begin{tabularx}{\textwidth}{XXX}\toprule
-        \textbf{Sync method} & \textbf{Pros} & \textbf{Cons} \\ \midrule
+\begin{table}[ht]
+    \begin{tabularx}{\textwidth}{lXX}\toprule
+        \textbf{Method} & \textbf{Pros} & \textbf{Cons} \\ \midrule
         GPS     & Precise clocks. & Limited to 1 Hz updates on Android. \\
-        Audio   & No external hardware requirements. & Very complicated with large number of devices, \newline large synchronization time, \newline audio lantency is different from device to device. \\
-        Light cues & No external hardware requirements. & Require cameras to face each other, \newline hardware limitations proces further complexity, \newline requires manual work. \\
-        \ac{NTP} & High-precition clocks, \newline mature protocol. & Requires internet. \\
+        Audio   & No external hardware requirements. & Very complicated with large number of devices, \newline large synchronization time, \newline audio latency is different from device to device. \\
+        Light cues & No external hardware requirements. & Require cameras to face each other, \newline hardware limitations process further complexity, \newline requires manual work. \\
+        \ac{NTP} & High-precision clocks, \newline mature protocol. & Requires internet. \\
         \ac{PTP} & Do not require internet, \newline accuracy in nanoseconds & Accuracy in nanoseconds only with cabled network \newline not very widespread among Android apps. \\ \bottomrule
     \end{tabularx}
     \caption{Pros and cons of the different synchronization methods.}\label{tab:pros_cons_sync}
 \end{table}
+
+
+
+\section{Refining the Synchronization Method --- SNTP}

```

Comment:
"make minor changes so as to improve or clarify (a theory or method)." - vil nu mene Refining passer meget godt på hvad der sker i denne section